apps/src/lib/tools/jsdebugger/MercatorMap.jsx

```diff
@@ -0,0 +1,68 @@
+// import PropTypes from 'prop-types';
+import React from 'react';
+// import {connect} from 'react-redux';
+import {geoMercator, geoPath} from 'd3-geo';
+import {feature} from 'topojson-client';
+
+const RADIUS = 20;
```

Comment:
Can we update these variable names to describe what the radius/width/height are for? (ex: "MERCATOR_MAP_WIDTH")

Reply:
Yes, I'll change it the variable names to be more descriptive.